uint8 groupMask
uint8 trajectoryId
float32 timescale
bool reversed
bool relative
---
